### ISSUE TITLE
Is it possible to search with larger image size like 224*224?

### ISSUE THREAD

**Issue by R00Kie-Liu** [quark0/darts#81]
I'm trying to search the cell with images resized to 224*224, but it seems that huge memory is required...

**Comment by Catosine** [comment]
Yes. You may need to 1) change the stem to reduce the size at the very beginning and 2) use smaller batch size.
